.github/workflows/main.yml

```diff
@@ -55,7 +58,6 @@ jobs:
         if: github.event_name == 'push'
         uses: chromaui/action@v1
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
token isn't documented as an option in the chromatic action so removing https://www.chromatic.com/docs/github-actions#available-options